python/ray/train/batch_predictor.py

```diff
@@ -306,6 +306,7 @@ def __call__(self, input_batch: DataBatchType) -> DataBatchType:
             if self.get_preprocessor() is not None
             else predict_stage_batch_format,
             batch_size=batch_size,
+            prefetch_batches=num_gpus_per_worker > 0,
```

Comment:
`prefetch_batches` should accept an integer, right?

Reply:
good point, updated!